config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/templates/cluster-api-provider-vsphere-periodics-upgrades.yaml.tpl

```diff
@@ -1,7 +1,17 @@
-{{ if eq $.branch "release-1.5" "release-1.6" "release-1.7" "release-1.8" | not }}
+{{ $modes := list "-govmomi" "-supervisor" -}}
+{{ range $i, $mode := $modes -}}
+{{ $modeFocus := "" -}}
+{{ if eq $mode "-supervisor" }}{{ $modeFocus = "\\\\[supervisor\\\\] " }}{{ end -}}
+{{/* e2e upgrade has been introduced in release-1.9 */ -}}
+{{/* e2e upgrade in supervisor mode has been introduced in release-1.10 */ -}}
+{{ $skipInBranch := list "release-1.5" "release-1.6" "release-1.7" "release-1.8" -}}
```

Comment:
If you want we can do one round of dropping release-1.5 in an additional commit (Otherwise another PR?)

Reply:
I would prefer do this in another PR (where we drop 1.5, add 1.10)